packages/core/src/mappings/Modal.js

```diff
@@ -0,0 +1,38 @@
+import {
+  GROUPS,
+  COMPONENT_TYPES,
+  FORM_TYPES,
+  PROP_TYPES,
+  createBoolProp,
+} from "@draftbit/types";
+
+export const SEED_DATA = {
+  name: "Modal",
+  tag: "Modal",
+  description: "A basic Modal Component",
+  category: COMPONENT_TYPES.container,
+  props: {
+    animationType: {
+      group: GROUPS.advanced,
```

Comment:
I already merged this, but I'm not actually seeing this prop anywhere. `Advanced` may be hidden, currently, so I think this needs to be `basic`.

Reply:
OK. I will create another PR to make them basic.